core/src/main/java/org/infinispan/container/impl/AbstractInternalDataContainer.java

```diff
@@ -52,6 +49,10 @@
 import org.infinispan.util.concurrent.CompletionStages;
 import org.infinispan.util.concurrent.WithinThreadExecutor;
 
+import com.github.benmanes.caffeine.cache.CacheWriter;
+import com.github.benmanes.caffeine.cache.Caffeine;
+import com.github.benmanes.caffeine.cache.RemovalCause;
```

Comment:
Are we ordering `com` imports after `org` imports now?

Reply:
That is just what IntelliJ did to me with the imported Infinispan settings file.

Reply:
Aha, it was me who moved them before :)